packages/angular_devkit/core/src/jobs/README.md

```diff
@@ -0,0 +1,493 @@
+# Description
+
+Jobs is the Angular DevKit subsystem for scheduling and running generic functions with clearly 
+typed inputs and outputs. A `Job` instance is code (ie. a function) that can be scheduled,
+synchronized and schedule other jobs. Jobs can be run in parallel or follow an execution
+strategy.
```

Comment:
The grammar of this sentence is still not right. -- "ie" means "for instance", which I think is not what you mean. The things a job can do are not parallel, so I can't tell if the job itself does the action, or if a controller does it.
Try:
"A `Job` instance is a function associated with metadata. You can schedule a job,
synchronize it with other jobs, and use it to schedule other jobs."
--or perhaps:
"...You can schedule a job, use a job to schedule other jobs, and synchronize scheduled jobs."

Reply:
As a quick side note, the literal translation of “id est” would be “that is”.  “For instance” is closer to a translation of “exempli gratia”.  Since “exempli” can be either an example or sample of something.

Reply:
You're right, of course - I was sloppy :-)
In any case, Google style tells us not to use Latin abbreviations, for exactly this reason.

Reply:
Done.